spotify_tensorflow/luigi/python_dataflow_task.py

```diff
@@ -0,0 +1,301 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2017 Spotify AB.
```

Comment:
nit: update year

Reply:
hmm... do we need to? I thought the year is when it starts (?)